.gitignore

```diff
@@ -0,0 +1,94 @@
+# Logs
```

Comment:
Added so `node_modules` doesn't explode the repo, and copypastad a bunch of node/sjango stuff as well